template/Makefile.j2

```diff
@@ -15,6 +15,9 @@ VERSION := $(shell cargo metadata --format-version 1 | jq '.packages[] | select(
 ## Docker related targets
 docker-build:
 	docker build --force-rm -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:${VERSION}" -f docker/Dockerfile .
+{[% if operator.product_string in ['opa'] %}]
```

Comment:
This is probably ok for now, but if we start adding more of these conditions here I think I would prefer adding a "dockerfiles" variable that defaults to `["docker/Dockerfile"]` but can be overridden to allow building and pushing multiple docker images here.